vassal-app/src/main/java/VASSAL/script/ExpressionInterpreter.java

```diff
@@ -687,14 +687,13 @@ private String unescape(String expr) {
   }
 
   private List<Map> getMapList(Object mapName, GamePiece sourcePiece) {
-    final List<Map> maps;
+    List<Map> maps = new ArrayList<>();
```

Comment:
Don't you want to avoid creating the `ArrayList` for the case where you're going to immediately throw it away?

Reply:
True. I have recoded getMapList, got rid of the temporary List alltogether